src/screens/IndexScreen.js

```diff
@@ -173,7 +198,7 @@ const getComponent = (query) => {
     [QUERY_TYPES.NEWS_ITEMS]: TextList,
     [QUERY_TYPES.POINTS_OF_INTEREST]: CardList,
     [QUERY_TYPES.TOURS]: CardList,
-    [QUERY_TYPES.CATEGORIES]: CategoryList
+    [QUERY_TYPES.CATEGORIES]: CategoryList,
```

Comment:
take care of the bad lintings please, that are not matching the rules of the project

Reply:
I'm going to take care of it today